homeassistant/components/tts/__init__.py

```diff
@@ -475,10 +498,17 @@ async def async_read_tts(self, filename):
             await self.async_file_to_mem(key)
 
         content, _ = mimetypes.guess_type(filename)
-        return content, self.mem_cache[key][MEM_CACHE_VOICE]
+        return content, cast(bytes, self.mem_cache[key][MEM_CACHE_VOICE])
```

Comment:
We're casting the second item in the tuple to bytes so then it can't be optional.

Reply:
I've adjusted the method definition to match this cast.